lib/broadcast/index.web.js

```diff
@@ -0,0 +1,42 @@
+const BROADCAST_ONYX = 'BROADCAST_ONYX';
+
+let subscriptions = [];
+const channel = new BroadcastChannel(BROADCAST_ONYX);
+
+/**
+ * Sends a message to the broadcast channel.
+ * @param {String} message
+ */
+function sendMessage(message) {
+    channel.postMessage(message);
+}
+
+/**
+ * Subscribes to the broadcast channel. Every time a new message
+ * is received, the callback is called.
+ * @param {Function} callback
+ */
+function subscribe(callback) {
+    subscriptions.push(callback);
+    channel.onmessage = (message) => {
+        subscriptions.forEach(c => c(message));
+    };
+}
+
+/**
+ * Unsubscribes from the broadcast channel.
+ */
+function unsubscribe() {
+    subscriptions = [];
```

Comment:
Just to clarify: should we unsubscribe from all channels, or a specific one?

Reply:
This is actually not being used, but it should might be required in a possible cleanup of Onyx. I added this here by completeness, but I can actually remove it.

Reply:
Let's keep it I think

Reply:
yeah, it is up to you